tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -104,18 +104,21 @@ module.exports = {
      *
      * @param {string} fileName
      */
-    deleteFile: async function (fileName) {
-      await this
+    deleteFile: function (fileName) {
+      return this
         .waitForFileVisible(fileName)
         .useXpath()
         .performFileAction(fileName, 'delete')
         .waitForElementEnabled(
           this.elements.deleteFileConfirmationBtn.selector,
           'css selector'
         )
-
+        .clickDeleteButton()
+    },
+    clickDeleteButton: async function () {
       const clickAction = function () {
-        return this.initAjaxCounters().click('@deleteFileConfirmationBtn')
+        return this.api.initAjaxCounters()
+          .useCss().click(this.elements.deleteFileConfirmationBtn)
```

Comment:
I think we can make it simpler by
```suggestion
          .click('@deleteFileConfirmationBtn')
```